server/package.json

```diff
@@ -8,19 +8,21 @@
     "dev": "nodemon src/server.js"
   },
   "dependencies": {
+    "bcryptjs": "^2.4.3",
     "dotenv": "8.1.0",
     "express": "4.17.1",
     "pg": "7.12.1",
     "pg-hstore": "2.3.3",
-    "sequelize": "5.18.4"
+    "sequelize": "^5.19.1"
   },
   "devDependencies": {
-    "cli": "1.0.1",
+    "cli": "^1.0.1",
```

Comment:
What does CLI do? Could you please remove it, I think it is not necessary. Also, please remove these `^` from all packages.